woocommerce/woocommerce
# Add sort order for migration script.

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds definitive sorting when migrating post orders to HPOS. Note that in most cases, migration will often be sorted on the trunk, as MySQL will try to sort based on whatever the first column is being selected. But this is not guaranteed and is not always the case. This PR adds definitive sort order to the migration scripts.



### How to test the changes in this Pull Request:




It's a minor non-UX change, however you can test it by (note that you may not able to replicate the non-sorting behavior on trunk) :

0. Delete HPOS tables on your test site if you have the migration done already by going to WooCommerce > Status > Tools
1. Start the HPOS migration from CLI, but the migration script midway.
2. Check that all the new migrated orders in the order tables are sorted.



### Other information:




### FOR PR REVIEWER ONLY:
